### PR TITLE
fix typo in protocal-spec_zh

### PR DESCRIPTION
**Pull request by jiangyaoguo**
fix typo in protocal-spec_zh.md
## Description

Fix typo and change some translation.
## Motivation and Context

Fixes #
## How Has This Been Tested?

No new test is required.
## Checklist:

- [x] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff).
- [x] I have either added documentation to cover my changes or this change requires no new documentation.
- [x] I have either added unit tests to cover my changes or this change requires no new tests.
- [x] I have run [golint](https://github.com/golang/lint) and have fixed valid warnings in code I have added or modified. This tool generates false positives so you may choose to ignore some warnings. The goal is clean, consistent, and readable code.

Signed-off-by: jiangyaoguo jiangyaoguo@gmail.com

### CODE REVIEW

File: docs/protocol-spec_zh.md [hunk 1]
```diff
@@ -1140,9 +1141,9 @@ type RemoteLedgers interface {
 ```
 func NewConsenter(cpi consensus.CPI) (consenter consensus.Consenter)
 ```
-这个函数读取为`peer`过程指定的`core.yaml`配置文件中的`peer.validator.consensus`的值。键`peer.validator.consensus`的有效值指定运行`noops`还是`obcpbft`共识。（注意，它最终被改变为`noops`或`custom`。在`custom`情况下，验证 peer 将会运行由`consensus/config.yaml`中定义的共识插件）
+这个函数读取为`peer`过程指定的`core.yaml`配置文件中的`peer.validator.consensus`的值。键`peer.validator.consensus`的有效值指定运行`noops`还是`pbft`共识插件。（注意，它最终被改变为`noops`或`custom`。在`custom`情况下，验证 peer 将会运行由`consensus/config.yaml`中定义的共识插件）
 
-插件的作者需要编辑函数体，来保证路由到它们包中正确的构造函数。例如，对于`obcpbft` 我们指向`obcpft.GetPlugin`构造器。
+插件的作者需要编辑函数体，来保证路由到它们包中正确的构造函数。例如，对于`pbft` 我们指向`obcpft.GetPlugin`构造器。
```
Comment (srderson) [inline docs/protocol-spec_zh.md]: `obcpft.GetPlugin` should be changed to `pbft.GetPlugin`

**Comment by srderson** [comment]
@mqshen @yeasy @GrapeBaBa @ckeyer Could you please assist with reviewing this change? Thanks

**Comment by yeasy** [comment]
LGTM!
Thanks for the correction!

**Comment by GrapeBaBa** [comment]
LGTM

**Comment by jiangyaoguo** [comment]
Thanks for the reviews. I have updated PR based on @srderson 's comment.

**Comment by mqshen** [comment]
LGTM!

**Comment by ckeyer** [comment]
LGTM

**Comment by jiangyaoguo** [comment]
I have migrating this PR to Gerrit at https://gerrit.hyperledger.org/r/#/c/343/.  Should I close this now?

**Comment by GrapeBaBa** [comment]
yes

**Comment by yeasy** [comment]
+1!

**Comment by jiangyaoguo** [comment]
Close this because migrating to Gerrit.

**Pull request closed without merging**